### DISCUSSION TITLE
Average telemetries

### DISCUSSION THREAD

**Discussion by McHouse** [theOehrly/Fast-F1#320] [Q&A]
A question from my ignorance, I'm new to python.
Can you average the telemetries of two or more laps of the same driver?
If so, how?

thanks.

**Comment by theOehrly** [comment]
With enough effort, you surely can do that. It will not be easy though, because of the somewhat inconsistent sample rate.
The question that poses itself is why you would event want to do that. I can think of no reasonable use case for this idea right now.

**Comment by dialtone** [comment]
You may want to calculate a lap made of the XY%ile of the car's speed around the track, or see what was the average speed of a few cars in a given set of laps on a straight. It's not that far fetched, it's not very easy though yeah.

**Reply by theOehrly** [reply]
True, that makes sense. But as said before, not easily done. It'll require a bit of experimentation to figure out what approach would give the best result. It's not something that I have tried before, so I can't really help.

**Reply by McHouse** [reply]
Thank you all for your instructions, although the final result does not "excite" me. hahaha

Indeed, my idea was to compare average speeds in specific sections, not just in a fast lap or just one lap.

Well, at least (sometimes) I don't have very crazy ideas. 

Thanks to everyone for your interest and help.

**Comment by McHouse** [comment]
Sorry for my answer in the wrong place.
It is clear that I am new to these "stuff"
